pkg/configschema/comments_test.go

```diff
@@ -12,31 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// skipping windows to avoid this golang bug: https://github.com/golang/go/issues/51442
-//go:build !windows
-
-package cfgmetadatagen
+package configschema
```

Comment:
Based on the file names, I guess this is just Github showing a weird diff, right? Or can we run this on Windows now for some reason?

Reply:
Yeah, maybe. I wanted to try running this on Windows and see what if any failures there were, but if I understood correctly from a comment made during the SIG meeting, Windows CI tests only run on the main branch? Going to see if I can get these to run, otherwise will revert to skipping Windows tests.

Reply:
I added the `Run Windows` label to run the Windows tests on this PR

Reply:
Hm, it doesn't look like it's working properly :thinking: I think we should revert to skipping tests to avoid breaking main